web_monitoring/cli/cli.py

```diff
@@ -605,10 +613,9 @@ def import_ia_urls(urls, *, from_date=None, to_date=None,
             maintainers,
             tags,
             stop_event,
-            unplaybackable,
+            unplaybackable=unplaybackable,
             tries=(None,
-                   dict(retries=3, backoff=4, timeout=(30.5, 2)),
-                   dict(retries=7, backoff=4, timeout=60.5))))
+                   dict(retries=3, backoff=4, timeout=(30.5, 2)))))
         memento_thread.start()
```

Comment:
I simplified this since the second and third retries haven’t actually been running for a while anyhow.